sqlparser/sql.go

```diff
@@ -618,572 +619,521 @@ var yyExca = [...]int{
 	153, 272,
 	-2, 262,
 	-1, 248,
-	111, 616,
+	111, 617,
```

Comment:
o.o

Reply:
this code is fully generated, right?

Reply:
sql.go is auto-generated from sql.y via goyacc